go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/containerd/go-cni
 
 require (
-	github.com/containernetworking/cni v0.8.0
+	github.com/containernetworking/cni v0.8.1
```

Comment:
not sure if it makes sense to move up to 0.8.1.. crio and k/k are at 0.8.0 and all I see there is a "tightening" of the plugin finding logic with no explanation

Reply:
Let me move the last commit from this PR